examples/onchain-signer/test/CommentBox.ts

```diff
@@ -65,18 +66,18 @@ describe('CommentBox', function () {
     ]);
 
     // Sapphire Mainnet/Testnet: Wait a few moments for nodes to catch up.
-    if (
-      (await gasless.provider.getNetwork()).chainId == 23294 ||
-      (await gasless.provider.getNetwork()).chainId == 23295
-    ) {
+    if (chainId == BigInt(23294) || chainId == BigInt(23295)) {
       await new Promise((r) => setTimeout(r, 6_000));
     }
 
-    const tx = await gasless.makeProxyTx(commentBox.address, innercall);
+    const tx = await gasless.makeProxyTx(
+      await commentBox.getAddress(),
+      innercall,
+    );
 
     // TODO: https://github.com/oasisprotocol/sapphire-paratime/issues/179
-    const response = await gasless.provider.sendTransaction(tx);
-    const receipt = await gasless.provider.waitForTransaction(response.hash);
+    const response = await ethers.provider.broadcastTransaction(tx);
+    const receipt = await ethers.provider.getTransactionReceipt(response.hash);
```

Comment:
This was marked as `pending` before... https://github.com/oasisprotocol/sapphire-paratime/actions/runs/7109069502/job/19353461591

Reply:
Red herring. This works with the fix for `await`.